static/app/components/assistant/getGuidesContent.tsx

```diff
@@ -273,6 +273,24 @@ export default function getGuidesContent(orgSlug: string | null): GuidesContent
         },
       ],
     },
+    {
+      guide: 'explain_new_default_event_issue_detail',
+      requiredTargets: ['issue_details_default_event'],
+      dateThreshold: new Date('2023-08-22'),
+      steps: [
+        {
+          title: t('New Default Event'),
+          target: 'issue_details_default_event',
+          description: tct(
+            'Rather than the latest event, we now default to an event with the most context (replays, traces, and profiles). You can easily switch between events or [link:configure your default event] in settings.',
```

Comment:
After reading this one more time, I think we should add that it is a _recent_ event as well. So maybe something like `we now default to a recent event with the most context`. Just so users know that is part of the criteria.